extensions.bzl

```diff
@@ -44,7 +44,7 @@ def _ext_impl(mctx):
         }
     if gcc_toolchain_tag:
         macro_args |= {
-            "gcc_major_version": gcc_toolchain_tag.gcc_major_version,
+            "gcc_version": gcc_toolchain_tag.gcc_version,
```

Comment:
This should be the correct macro args, but i think the extension arg should be kept the same. We dont need to keep the arg backward consistent with the macro. Major version should properly communicate to users that only the major version is used in the include/import path in the toolchain

Reply:
Seems kind of silly to have 2 separate names for the same thing

Reply:
> Seems kind of silly to have 2 separate names for the same thing

yeah, I agree... is it possible to change it?

If I've understood things correctly, this repo is `development` only so far and it's still not "public" with a stable version, etc, right? How many use-cases are currently using the macro? Is it OK if it breaks?

I'm happy to change it all to `gcc_major_version`, just let me know what you prefer.